core/src/components/textarea/test/fill/index.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
```

Comment:
Threading for discussion:

Thoughts on increasing the height of the textarea? The textarea height is so small that it looks like the text is off center. If you compare with https://material-components.github.io/material-components-web-catalog/#/component/text-field, the height is larger so the text does not look off center.

Reply:
Looking at https://material-components.github.io/material-components-web-catalog/#/component/text-field a  bit more, when you remove the height from the MD textarea, the dimensions and positioning look the same as their text field.

| default textarea | textarea with height removed |
| - | - |
| ![image](https://user-images.githubusercontent.com/2721089/213580324-c019d27f-8f50-4c32-8814-741bef5ecd12.png) | ![image](https://user-images.githubusercontent.com/2721089/213580388-02451b58-5d7f-4756-8317-7ddb03a924d2.png) |

I wonder if we'd have more luck if we tried this:

1. Revert textarea to have the same padding/margins as the new input syntax.
2. Make the textarea taller by default so it visually stands out when compared to the input.